tests/rptest/services/rpk_producer.py

```diff
@@ -25,18 +26,24 @@ def __init__(self,
         self._acks = acks
         self._printable = printable
         self._stopping = Event()
+        self._quiet = quiet
 
     def _worker(self, _idx, node):
         rpk_binary = self._redpanda.find_binary("rpk")
         key_size = 16
         cmd = f"dd if=/dev/urandom bs={self._msg_size + key_size} count={self._msg_count}"
+
         if self._printable:
             cmd += ' | hexdump -e "1/1 \\"%02x\\""'
 
         cmd += f" | {rpk_binary} topic --brokers {self._redpanda.brokers()} produce --compression none {self._topic} -f '%V{{{self._msg_size}}}%K{{{key_size}}}%k%v'"
         if self._acks is not None:
             cmd += f" --acks {self._acks}"
 
+        if self._quiet:
```

Comment:
Maybe you should not do `self.logger.debug(line.rstrip())` for line, if you set _quiet?

Reply:
The idea of quiet mode is just to avoid the heavy per-message output -- if there is other output like errors, I'd still like to see it in the ducktape log.